### DISCUSSION TITLE
Dropped packet

### DISCUSSION THREAD

**Discussion by CyrilFRA** [chmorgan/sharppcap#444] [Q&A]
Hello,

I started a test on a custom protocol acquirement, with a packet every 1,5ms. There were no problem for 5 days without interruption.
On the 6th day (at 2:00 AM, when everyone's at home), I had a Dropped issues :
- Dropped Packets : 1283
- Interface Dropped Packets : 0

I'm pretty sure my program is reading packets on time... (I use callback method)

I don't understand what's going on, do you have an idea ?

Best Regards

**Comment by kayoub5** [comment]
Usually that happens when PC is doing something that slows down the whole system (scheduled updates, anti-virus background scan)

Usually increasing the buffer size in Open() configuration reduces the issue.

**Comment by CyrilFRA** [comment]
Hello @kayoub5, thanks for your answer,
It's strange, because the PC I use is off WAN (so no updates), and doesn't have anti-virus (to keep performances).
I'll try to increase the buffer size. **Do you what is the default value ?** (maybe 1MB but I'm not sure...)
Best Regards

**Reply by CyrilFRA** [reply]
I have another question : what is the difference between the BufferSize and KernelBufferSize configuration ? The first uses pcap_set_buffer_size() while the second uses pcap_setbuff(), but I didn't get the difference between the 2 ways of configuration... Is it the save ?
Which one should I use ?
